world-explore/src/app/home/home.page.ts

```diff
@@ -86,20 +89,16 @@ export class HomePage implements OnInit {
     }
   }
 
-  toggleFavorite(country: Country,event :any) {
-    console.log(event.detail)
-    if (event.detail.side=="start")
-      {
-    if (!this.favoriteCountries.has(country.country)) {
-      this.favoriteCountries.add(country.country);
+  toggleFavorite(country: Country, event: any) {
+    if (event.detail.side === "start") {
```

Comment:
1-  Check if the swipe event is from the start.
2- Check if the country is not already in the favoriteCountries set
3- Use ternary operator to perform action based on conditions
4- Update the list of filtered countries after modifying favorites